docs/content/developers/storage-to-rest-patterns.md

```diff
@@ -1,12 +1,14 @@
-### workspaces
+# Storage to REST Patterns
 
-kcp promises to support 1 mln of workspaces. 
-A workspace is like a Kubernetes endpoint, i.e. an endpoint usual Kubernetes client tooling (client-go, controller-runtime and others) 
+## Logical Clusters
+
+kcp promises to support 1 mln of logical clusters. 
```

Comment:
nit: I don't think this is the correct abbreviation. I think it's either "M" or super formal "MM". Nonetheless I think I would prefer to just write out million.